src/services/depositService.ts

```diff
@@ -283,11 +306,15 @@ export function getAvailableDepositRoutes(
     case ChainType.EVM:
       switch (network) {
         case BlockchainEnum.NEAR:
-        case BlockchainEnum.TURBOCHAIN:
           return {
             activeDeposit: false,
             passiveDeposit: false,
           }
+        case BlockchainEnum.TURBOCHAIN:
+          return {
+            activeDeposit: true,
```

Comment:
So we'll have two types of active deposits. One is deposit from a silo, another is transfer using a bridge. Do you have plans how to implement it?

Reply:
This can be just a generated address, just like a `passiveDeposit`, and the user will need to send the token there themselves, or we can add a variable, something as `bridgeDeposit`.